common/fabhttp/server_test.go

```diff
@@ -71,7 +71,7 @@ var _ = Describe("Server", func() {
 	When("trying to connect with an old TLS version", func() {
 		BeforeEach(func() {
 			tlsOpts := []func(config *tls.Config){func(config *tls.Config) {
-				config.MaxVersion = tls.VersionTLS11
+				config.MaxVersion = tls.VersionTLS12
```

Comment:
this is a test that forces to use TLS 1.1. It is called "trying to connect with an old TLS version" :-)

Reply:
So it doesn't make sense here ?

Reply:
Doesn't make sense here, the intent of the test is to use the old TLS version. Closing...